src/main/java/htsjdk/samtools/fastq/FastqRecord.java

```diff
@@ -228,8 +228,13 @@ public String toFastQString() {
     }
 
     /**
-     * Returns {@link #toFastQString()}
+     * Returns {@link #toFastQString()}.
+     *
+     * @deprecated since 07/2017. This method will be deprecated until it is changed in the future
+     * for a simpler representation of the object. For code relying on the formatting as a FASTQ
+     * String, please refactor your code to use {@link #toFastQString()}.
      */
+    @Deprecated
```

Comment:
I believe that deprecating concrete implementation of `Object.toString` is a good way to warn in direct uses of the method for developers; if they are using the `toString` only from the object perspective, they shouldn't get a warning.

If you have any proposal on how to warn users about this to be able to change the behaviour in the next one/two releases, I can change the deprecation of the class in this PR.